### DISCUSSION TITLE
从PAG官网https://pag.art/下载的PAGViewer Mac版，安装完成之后打开crash闪退?

### DISCUSSION THREAD

**Discussion by yufengvac** [Tencent/libpag#2213] [Q&A] [answered]
<img width="988" alt="image" src="https://github.com/Tencent/libpag/assets/7565523/ac6073c6-43e9-4740-88e2-1e7b506aee1b">

[pagviewer崩溃日志.txt](https://github.com/Tencent/libpag/files/14663288/pagviewer.txt)

**Comment by wangwanjie** [comment]
> <img alt="image" width="988" src="https://private-user-images.githubusercontent.com/7565523/314413488-ac6073c6-43e9-4740-88e2-1e7b506aee1b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTIwMjg1ODEsIm5iZiI6MTcxMjAyODI4MSwicGF0aCI6Ii83NTY1NTIzLzMxNDQxMzQ4OC1hYzYwNzNjNi00M2U5LTQ3NDAtODhlMi0xZTdiNTA2YWVlMWIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDQwMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA0MDJUMDMyNDQxWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NTYyZjYwMDJkMTZhNzg2ZDQ4OGU5NzA5MmRlYTM5MDBjN2YzZmU3NThiNzMzMjE2MzUwYTlkYjM2NTc1ZWUwNyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.eHYs0OaHx6K7sRj7jRv844oLVUh95GDEjm0TaHAMYEQ">
> [pagviewer崩溃日志.txt](https://github.com/Tencent/libpag/files/14663288/pagviewer.txt)

下载历史版本
https://cdn.pag.art/pagart/file/PAGViewer-4.0.5-stable-macOS.dmg

**Comment by zhalice2011** [comment, marked as answer]
你好, 安装一下历史版本试试

https://pag.art/docs/pag-download.html
